### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Authentication_Screen Reader: Narrator is not reading the title of the PopUp

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#91]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #ScreenReader; #Narrator; #Chrome; #NVDA; #MAS1.3.2

**Environment Details:**
URL: https://manage.iis.net/webserver/authentication
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome 68.0.3440.106
Screen Reader: Narrator, NVDA
MAS ID Violated: MAS 1.3.2

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Authentication and press enter
6. Tab till windows tab and press enter
7. Turnoff the first toggle button in the page
8. Verify narrator is reading the title of the popup or not

**Actual Result:**
Narrator is not reading the title of the popup

**Expected Result:**
Narrator should read the title of the popup

**Note:**
This issue has been observed in all the popups

**User Impact:**
Users with disabilities finds its difficult in working through the application if narrator doesnot read the title of the popup

**Attachments:**
[Bug91_Web server_Authentication_Narrator not reading title of the popup.pptx](https://github.com/Microsoft/IIS.WebManager/files/2322796/Bug91_Web.server_Authentication_Narrator.not.reading.title.of.the.popup.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
